quickstart/quickstart.yaml

```
@@ -558,7 +558,7 @@ spec:
   description: "API Demo Portal"
   apiGateway: traefik-hub-demo-gateway
   ui:
-    logoUrl: https://doc.traefik.io/traefik-hub/assets/img/traefik-hub-logo.svg
+    logoUrl: https://doc.traefik.io/traefik-hub/img/hub-logo-light.svg
```

I'd keep using the square logo if possible:

```suggestion
    logoUrl: https://doc.traefik.io/traefik-hub/img/traefik-hub-logo.svg
```